ci fails with WebDriver.__init__() got multiple values for argument 'options'

got this:
```
Traceback (most recent call last):
  File "//screenshot.py", line 259, in <module>
    main()
  File "//screenshot.py", line 95, in main
    screenshot(url_to_open, png_path)
  File "//screenshot.py", line 108, in screenshot
    with _get_driver() as driver:
         ^^^^^^^^^^^^^
  File "//screenshot.py", line 243, in _get_driver
    driver = webdriver.Chrome(ChromeDriverManager().install(), options=wd_options)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: WebDriver.__init__() got multiple values for argument 'options'
Error: Process completed with exit code 1.
```
moving target moved underneath us. Others are affected, too: https://stackoverflow.com/questions/76428999/selenium-chrome-webdriver-stopped-working

--- a/ci/screenshot.py
+++ b/ci/screenshot.py
@@ -30,6 +30,7 @@ import time
 import selenium.common.exceptions
 from selenium import webdriver
 from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.chrome.service import Service
 from selenium.webdriver.common.by import By
 from webdriver_manager.chrome import ChromeDriverManager
 
@@ -240,7 +241,9 @@ def _get_driver():
 
     log.info("set up chromedriver with capabilities %s", wd_options.to_capabilities())
 
-    driver = webdriver.Chrome(ChromeDriverManager().install(), options=wd_options)
+    driver = webdriver.Chrome(
+        service=Service(ChromeDriverManager().install()), options=wd_options
+    )
     # The waiter is only optionally used later.
     # waiter = WebDriverWait(driver, 30)
     # Piggy-back waiter object on driver object, use a somewhat unique name

